confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -311,7 +312,7 @@ def get_compatibility(self, subject=None):
         if code >= 200 and code <= 299:
             compatibility = result.get('compatibility', None)
 
-        if not compatibility:
+        if compatibility is None:
```

Comment:
This changes the behavior if `result.get('compatibility', None)` returns an empty string, but I assume that's not possible